### DISCUSSION TITLE
Help using the plugin and configuring the theme/segments

### DISCUSSION THREAD

**Discussion by farzadmf** [erikw/tmux-powerline#453] [Q&A]
Hi, I installed the plugin, and everything works fine.

However, I have issues trying to configure things: I basically want to configure the segments I see

I created `~/.config/tmux-powerline/config.sh` with the following content:
```
export TMUX_POWERLINE_THEME="bubble"
```
This works fine, and I see the theme change.

However, if I do `cp ~/<tmux-powerline-installed-by-tpm-path>/themes/bubble.sh ~/.config/tmux-powerline/themes/my.sh` and change the above line to say
```
export TMUX_POWERLINE_THEME="my"
```
everything gets messed up.

When I restart `tmux`, after a delay, I see a message saying `~/.tmux/plugins/tpm/tpm returned 1`

I'd appreciate any help please

**Comment by erikw** [comment]
Hello, did you solve this? Sorry I must have missed the notification for this one.

**Reply by farzadmf** [reply]
@erikw thank you for following up.

It's been quite a while, so I've mostly forgotten what was happening at the time.

But IIRC, I gave up doing things "the proper" way and ended up copying the whole config or something like that.

I did create #452 at the time as well (you might remember), but I missed your reply(es) on that (I myself HATE people opening issues and disappearing and I did it myself 😔 )

UPDATE: yeah, I just checked what I'm doing and I think I basically copy/pasted what `tmux-powerline` provides as theme and comment/change the values to have my theme, and then I override default `tmux-powerline` files with those files.

It's far from ideal IMO, but since I did want to use this plugin and the "proper" way didn't work, I ad to come up with a workaround
